views/agenda.tmpl

```diff
@@ -4,7 +4,7 @@
     <link href="/css/charts.css" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/dygraph/2.1.0/dygraph.min.css" />
     {{template "html-head" printf "Decred Vote Agenda Pages"}}
-    <body>
+    <body class="{{ .Theme }}">
```

Comment:
OK, but consider that you could avoid the `Theme` struct field in the page's data by doing something like `{{print "theme-" .NetName}}`

Reply:
There could even be `{{$Theme := print "theme-" .NetName}}` in the header I think.

Reply:
Well, I've neglected `ToLower`, but there are several ways to do this anyway.   You can add to the `template.FuncMap`:

```go
"lower": func(s string) string {
	return strings.ToLower(s)
},
```

or cut to the chase and edit `makeTemplateFuncMap`:

```go
func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
	netTheme := "theme-" + strings.ToLower(netName(params))
	return template.FuncMap{
...
		"theme": func() string {
			return netTheme;
		}
...
```

That would capture the netTheme for the current network in the closure so that it is only generated once, and it only needs to be in this one place instead of several places in each page handler.